test_component.py

```diff
@@ -63,9 +63,10 @@ def oncounter(counter, id, type):
 
             # CALL
             try:
-                res = yield self.call(u'com.example.add', x, 3)
+                res = yield self.call(u'com.example.add2', x, 3)
```

Comment:
This https://github.com/crossbario/autobahn-java/blob/master/demo-gallery/src/main/java/io/crossbar/autobahn/demogallery/netty/Service.java#L117